android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -131,47 +130,43 @@ private void loadEvent(Event event) {
                 DateUtils.formatDateWithDefault(DateUtils.FORMAT_DATE_COMPLETE, event.getStartTime()),
                 DateUtils.formatDateWithDefault(DateUtils.FORMAT_DATE_COMPLETE, event.getEndTime()));
 
-        welcomeMessage.setText(getText(R.string.welcome_message));
-        Views.setHtml(organiser_description, event.getOrganizerDescription(), true);
-        Views.setHtml(event_description, event.getDescription(), true);
-        venue_details.setText(event.getLocationName());
-        event_timing.setText(date);
+        welcomeMessage.setText(getText(R.string.welcome_message) + " " + event.getName());
```

Comment:
This will cause localisation issues in the future. It is better to use `String.format` for this. Please refer this for help - https://stackoverflow.com/a/24098427